src/app/find-help/view-offer-accommodation/view-offer-accommodation.component.ts

```diff
@@ -37,6 +37,10 @@ export class ViewOfferAccommodationComponent implements OnInit {
       .catch((e) => console.error(e));
   }
 
+  getListUrl() {
```

Comment:
maybe getGoBackToListUrl?

Reply:
in a fact it is getUrlWithoutLastPathElement if I understand regex correctly, feel free to take the name if you like it

Reply:
`getUrlWithoutLastPathElement` describes the implementation and I care about the intent more. For example, if routing changed and I had to cut off two last URL parts, I'd just change the implementation but the method name should stay the same.

The fact that I assign it to the "Go back to list" link doesn't mean I cannot use this method for something else, e.g. to construct a new URL starting with the list one. That's why I chose `getListUrl`.